test/e2e/specs/specs-playwright/wp-signup__paid.ts

```diff
@@ -10,174 +10,183 @@ import {
-			expect( newAmount ).toBeGreaterThan( originalAmount );
-		} );
-
-		it( 'Enter billing and payment details', async function () {
-			const paymentDetails = DataHelper.getTestPaymentDetails();
-			await cartCheckoutPage.enterBillingDetails( paymentDetails );
-			await cartCheckoutPage.enterPaymentDetails( paymentDetails );
-		} );
-
-		it( 'Make purchase', async function () {
-			await cartCheckoutPage.purchase();
-		} );
-	} );
-
-	describe( 'Launch site', function () {
-		it( 'Navigate to Home dashboard', async function () {
-			const navbarComponent = new NavbarComponent( page );
-			await navbarComponent.clickMySites();
-		} );
-
-		it( 'Verify site is not yet launched', async function () {
-			// Obtain a new Page in a separate BrowserContext.
-			const testContext = await BrowserManager.newBrowserContext();
-			const testPage = await BrowserManager.newPage( { context: testContext } );
-			// TODO: make a utility to obtain the blog URL.
-			await testPage.goto( `https://${ blogName }.wordpress.com` );
-			// View site without logging in.
-			const comingSoonPage = new ComingSoonPage( testPage );
-			await comingSoonPage.validateComingSoonState();
-			// Dispose the test page and context.
-			await BrowserManager.closePage( testPage, { closeContext: true } );
-		} );
-
-		it( 'Start site launch', async function () {
-			const sidebarComponent = new SidebarComponent( page );
-			await sidebarComponent.navigate( 'Settings', 'General' );
-			const generalSettingsPage = new GeneralSettingsPage( page );
-			await generalSettingsPage.launchSite();
-		} );
-
-		it( 'Skip domain purchasse', async function () {
-			const domainSearchComponent = new DomainSearchComponent( page );
-			await domainSearchComponent.clickButton( 'Skip Purchase' );
-		} );
-
-		it( 'Confirm site is launched', async function () {
-			const myHomePage = new MyHomePage( page );
-			await myHomePage.validateTaskHeadingMessage( 'You launched your site!' );
-		} );
-	} );
-
-	describe( 'Cancel plan', function () {
-		it( 'Navigate to Upgrades > Plans', async function () {
-			const sidebarComponent = new SidebarComponent( page );
-			await sidebarComponent.navigate( 'Upgrades', 'Plans' );
-		} );
-
-		it( 'Manage plan', async function () {
-			const plansPage = new PlansPage( page );
-			await plansPage.clickTab( 'Plans' );
-			await plansPage.clickPlanActionButton( { plan: 'Personal', buttonText: 'Manage plan' } );
-		} );
-
-		it( 'Opt to cancel and refund', async function () {
-			const individualPurchasePage = new IndividualPurchasePage( page );
-			await individualPurchasePage.cancelPurchase();
-		} );
-	} );
-
-	describe( 'Delete user account', function () {
-		it( 'Close account', async function () {
-			const closeAccountFlow = new CloseAccountFlow( page );
-			await closeAccountFlow.closeAccount();
-		} );
-	} );
-} );
+const isStagingOrProd = DataHelper.getCalypsoURL()
+	.toLowerCase()
+	.includes( 'https://wordpress.com' );
+
+describeif( ! isStagingOrProd )(
```

Comment:
>I think it would make sense to use the @quarantine group

Hmm, can you explain that a little bit more? I don't think there's any way to optionally include a test in a given jest group based on something at runtime unfortunately.

To help clarify a little more: I think these tests are ready to run as part of pre-release tests, and I definitely think they should! The catch though is that this new flow is only turned on in `wpcalypso` config file, so you have to hitting the `wpcalypso.wordpress.com` (or Calypso.live) host to see it! One day they'll add it to the staging config and then we can take that check. But until then, I think we could get a lot of benefit running these in pre-release!

Does that help explain the check a bit more?

Reply:
> The catch though is that this new flow is only turned on in wpcalypso config file

I think that's the part I was missing and now that you've clarified that point, it makes sense why we are using the conditionals.